app/controllers/brexit_checker_controller.rb

```diff
@@ -1,5 +1,8 @@
 class BrexitCheckerController < ApplicationController
   include BrexitCheckerHelper
+
+  SUBSCRIBER_LIST_GROUP_ID = "5a7c11f2-e737-4531-a0bc-b5f707046607".freeze
```

Comment:
Nit: Feels like an awkward place for this config. Maybe in future it'd be good to have a class dedicated to email subscriptions and putting it under that namespace.

Reply:
Yup, I know what you mean, a controller feels a weird place but that's where we're interacting with the API.

I could have put it on BrexitCheckerHelper but then that obfuscates it away.